dt-contacts/contacts-post-type.php

```diff
@@ -271,46 +286,104 @@ public function get_contact_field_defaults( $post_id = null, $include_current_po
+                  "label" => __( 'Reading Bible', 'disciple_tools' ),
+                  "description" => _x("This contact is reading the bible.", "Reading Bible", 'disciple_tools') ],
+                "milestone_belief"        => [
+                  "label" => __( 'States Belief', 'disciple_tools' ),
+                  "description" => _x("This contact states belief. i.e. they have repented and believed.", "States Belief", 'disciple_tools'), ],
+                "milestone_can_share"     => [
+                  "label" => __( 'Can Share Gospel/Testimony', 'disciple_tools' ),
+                  "description" => _x("This contact has been trained to share the Gospel and their testimony.", "Can Share Gospel/Testimony", 'disciple_tools'), ],
+                "milestone_sharing"       => [
+                  "label" => __( 'Sharing Gospel/Testimony', 'disciple_tools' ),
+                  "description" => _x("This contact is sharing the Gospel and their testimony.", "Sharing Gospel/Testimony", 'disciple_tools'), ],
+                "milestone_baptized"      => [
+                  "label" => __( 'Baptized', 'disciple_tools' ),
+                  "description" => _x("This contact has been baptized.", "Baptized", 'disciple_tools'), ],
+                "milestone_baptizing"     => [
+                  "label" => __( 'Baptizing', 'disciple_tools' ),
+                  "description" => _x("This contact is baptizing others.", "Baptizing", 'disciple_tools'), ],
+                "milestone_in_group"      => [
+                  "label" => __( 'In Church/Group', 'disciple_tools' ),
+                  "description" => _x("This contact is in a church or group.", "In Church/Group", 'disciple_tools'), ],
+                "milestone_planting"      => [
+                  "label" => __( 'Starting Churches', 'disciple_tools' ),
+                  "description" => _x("This contact is planting Churches.", "Starting Churches", 'disciple_tools'), ],
             ],
             "customizable" => "add_only"
         ];
 
-
         $fields['baptism_date'] = [
             'name'        => __( 'Baptism Date', 'disciple_tools' ),
-            'description' => 'The date the contact was baptised',
+            'description' => __( 'The date the contact was baptised. For metrics reporting, it is important to always note the day a person is baptised.', 'disciple_tools' ),
             'type'        => 'date',
             'default'     => '',
             'section'     => 'misc',
         ];
 
         $fields['baptism_generation'] = [
             'name'        => __( 'Baptism Generation', 'disciple_tools' ),
-            'description' => 'The generation of baptism',
+            'description' => __( 'The generation of baptism.', 'disciple_tools' ),
             'type'        => 'text',
             'default'     => '',
             'section'     => 'misc',
         ];
 
         // Misc Information fields
-
+        $fields["contact_name"] = [
```

Comment:
@mikeallbutt please don't add extra fields. This will create confusion in other places.